interfaces/builtin/timezone_control.go

```diff
@@ -59,6 +59,13 @@ dbus (send)
     member="SetTimezone"
     peer=(label=unconfined),
 
+dbus (send)
+    bus=system
+    path=/org/freedesktop/timedate1
+    interface=org.freedesktop.timedate1
+    member="ListTimezones"
+    peer=(label=unconfined),
```

Comment:
I'd also add `name=org.freedesktop.timedate1`, though it's unlikely that some other service will register a `/org/freedesktop/timedate1` just for fun.

Reply:
none of the other dbus permissions define ```name=```, while i do not mind adding it, should we then add it to all other dbus definitions in the code (and for consistency also to the time-control and timeserver-control interfaces that copy these dbus definitions verbatim) ?

Reply:
Given that all D-bus traffic is mediated by the D-Bus server, which will not forward a request unless the destination has previously registered an object path at the right location, I don't see any security risk in omitting the name.
I'll wait to see what other team members say (in any case we need two +1, IIRC), then we can resolve this either way.

Reply:
I don't see much value in adding the `name` qualifier here since as @mardy said the `path` is the most important part here.